docs/user/document/distribution/tlp/label-usage.en.md

```diff
@@ -1,22 +1,22 @@
 # Label - Usage
 
-* The label should be set and must be valid. There are four color (or traffic light) values:
+The label should be set and be valid. There are four color (or traffic light) values:
 
-  * `RED` - *personal for named recipients only*:
+* `RED` - *personal for named recipients only*:
 
-    In the context of a meeting, for example, RED information is limited to those present at the meeting.
-    The distribution of RED information will generally be via a defined list and in extreme circumstances may only be passed verbally or in person.
+  In the context of a meeting, for example, RED information is limited to those present at the meeting.
+  The distribution of RED information will generally be via a defined list and in extreme circumstances may only be passed verbally or in person.
```

Comment:
It need the `TLP:` part upfront (applies to all).
```suggestion
  The distribution of `TLP:RED` information will generally be via a defined list and in some circumstances may only be passed verbally or in person.

Reply:
updated